app/src/components/code/JSONEditor.tsx

```diff
@@ -30,12 +30,17 @@ export function JSONEditor(props: JSONEditorProps) {
   const { theme } = useTheme();
   const { jsonSchema, ...restProps } = props;
   const codeMirrorTheme = theme === "light" ? githubLight : nord;
+  const [key, setKey] = React.useState(0);
+  useEffect(() => {
+    setKey((prev) => prev + 1);
+  }, [jsonSchema, props.value]);
```

Comment:
Pretty sure this would force a re-mount on every keystroke right?

Reply:
yeah was testing updated and removed!